flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -195,10 +195,14 @@ class SentryFlutterOptions extends SentryOptions {
   /// Only attach a screenshot when the app is resumed.
   bool attachScreenshotOnlyWhenResumed = false;
 
+  @Deprecated(
+      'Will be removed in a future version. Use [beforeCapture] instead')
+  BeforeScreenshotCallback? beforeScreenshot;
+
   /// Sets a callback which is executed before capturing screenshots. Only
   /// relevant if `attachScreenshot` is set to true. When false is returned
   /// from the function, no screenshot will be attached.
-  BeforeScreenshotCallback? beforeScreenshot;
+  BeforeCaptureCallback? beforeCapture;
```

Comment:
should it not be `beforeScreenshotCapture`? to make it a bit more specific

see: [here](https://docs.sentry.io/platforms/android/enriching-events/screenshots/#customize-screenshot-capturing) and [here](https://github.com/getsentry/sentry-java/blob/a1831635b97fd324c127bc4857fc307171d8394b/sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroidOptions.java#L497C1-L507C1)

Reply:
On Android it is just `beforeCapture` as it is also used before capturing view hierarchy. Wanted to have the same naming and also when we introduce the call before doing view hierarchy, we don't have to deprecate/rename again.

Reply:
<img width="396" alt="image" src="https://github.com/user-attachments/assets/190506a3-3e04-497e-9019-dd389059d5a6">

on Android there are two different options  whereas in this PR there is only one `beforeCapture` or am I missing something, the `SentryAndroidOptions.java` class also confirms this

Reply:
Ah my bad, assumed there is one callback because they use the same type. Will update.